### PR TITLE
Changes made for handling edge cases where file may not exist or be empty

### PR DESCRIPTION
**Pull request by aru31**
This is a fix for #266

### CODE REVIEW

File: autowebcompat/utils.py [hunk 1]
```diff
@@ -188,13 +188,13 @@ def make_infinite(gen_func, elems):
 
 
 def read_labels(file_name='labels.csv'):
-    try:
+    if os.stat(file_name).st_size == 0:
+        raise ValueError('labels file is empty')
```
Comment (marco-c) [inline autowebcompat/utils.py]: Raise a normal Exception, as ValueError has a specific meaning (`Raised when an operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as IndexError.`).

**Comment by aru31** [comment]
@marco-c i don't understand why coverage bot checks are failing. Please help

**Comment by marco-c** [comment]
> @marco-c i don't understand why coverage bot checks are failing. Please help

It's because your change is apparently not covered by tests, but it's fine, those checks are not required.

**Comment by marco-c** [comment]
Closing in favor of https://github.com/marco-c/autowebcompat/pull/269.

Please update the PR instead of opening a new one next time :)

**Pull request closed without merging**